lib/mpl_toolkits/tests/test_mplot3d.py

```diff
@@ -1002,6 +1002,16 @@ def test_lines_dists():
     ax.set_ylim(0, 300)
 
 
+def test_lines_dists_nowarning():
+    # Smoke test to see that no RuntimeWarning is emitted when two first
```

Comment:
would it be preferable to test a user-facing API rather than the private methods?

Reply:
This is mostly called through mouse callbacks so it is annoying to get public API tests that exercise this.